spras/omicsintegrator2.py

```diff
@@ -22,6 +22,7 @@
 """
 class OmicsIntegrator2(PRM):
     required_inputs = ['prizes', 'edges']
+    dois = []
```

Comment:
[Does oi2 have a paper?]

Reply:
No, only the github and I think it is used on the frankel website.

https://fraenkel-nsf.csbi.mit.edu/omicsintegrator/